Resources/Prototypes/_Gabystation/Guidebook/sop.yml

```
@@ -52,3 +72,63 @@
   name: guide-entry-pop-marcial
   text: "/ServerInfo/_Gabystation/Guidebook/POPMarcial.xml"
   priority: 6
+
+- type: guideEntry
+  id: POP Multa
+  name: guide-entry-pop-multa
+  text: "/ServerInfo/_Gabystation/Guidebook/POPMulta.xml"
+  priority: 7
+
+# POP Departamental
+
+- type: guideEntry
+  id: POP Comando
+  name: guide-entry-pop-comando
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Comando/CommandStandardOperatingProcedure.xml"
+  priority: 2
+
+- type: guideEntry
+  id: POP Segurança
+  name: guide-entry-pop-seguranca
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Security/SecurityStandardOperatingProcedure.xml"
+  priority: 3
+  children:
+  - POP Codigo de Conduta
+
+- type: guideEntry
+  id: POP Cargo
+  name: guide-entry-pop-cargo
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Cargo/CargoSOP.xml"
+  priority: 4
+
+- type: guideEntry
+  id: POP Engenharia
+  name: guide-entry-pop-engenharia
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Engineering/EngineeringSOP.xml"
+  priority: 5
+
+- type: guideEntry
+  id: POP Medico
+  name: guide-entry-pop-medico
+  text: "/ServerInfo/_Gabystation/Guidebook/POP/Medical/MedicalStandardOperatingProcedure.xml"
```

```suggestion
  text: "/ServerInfo/_Gabystation/Guidebook/POP/Medic/MedicalStandardOperatingProcedure.xml"
```
ou mudar no nome da pasta para `Medical` ou outra coisa e atualizar aqui